src/components/Comments.js

```diff
@@ -26,4 +27,9 @@ function Comments(props) {
     </div>);
 }
 
-export default Comments;
+const mapStateToProps = function (state) {
+  return {
+    Comments: state.Comments
```

Comment:
Think more about this part.  The left side of the colon is suppose to be the name of the prop this component is using. Is it using a prop called Comments?